README.md

```diff
@@ -15,11 +15,11 @@ The available commands are:
 
 `tsv`: transform a directory of parquet files into a directory of tsv files (which you can concat properly later with `hadoop fs -getmerge /path/to/tsvs`)
 
-    $ herringbone tsv -i /path/to/input/directory -o /path/to/output/directory
+    $ herringbone tsv -i /path/to/input/directory
```

Comment:
Can we actually show this as optional instead of removing it entirely? Maybe something like: `herringbone tsv -i /path/to/input/directory [-o /path/to/non/default/output/directory]`